resources/js/views/settings/roles/Index.vue

```diff
@@ -134,9 +138,9 @@ export default {
   computed: {
     tableFields () {
       const fields = [
-        { key: 'id', label: this.$t('settings.roles.id'), sortable: true },
+        { key: 'id', label: this.$t('settings.roles.id'), sortable: true, tdClass: 'td-max-width-0-lg', thStyle: { width: '8%' } },
         { key: 'name', label: this.$t('settings.roles.name'), sortable: true },
-        { key: 'default', label: this.$t('settings.roles.default'), sortable: true }
+        { key: 'default', label: this.$t('settings.roles.default'), sortable: true, tdClass: 'td-max-width-0-lg', thStyle: { width: '15%' } }
```

Comment:
why do you use the `td-max-width-0-lg` when the text truncate is not used in the cell?